src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -1176,6 +1217,32 @@ private string ResolvePSPath(string path)
             return result;
         }
 
+        /// <summary>
+        /// Gets the next literal path.
+        /// Filtering to ones that exist for the filesystem.
+        /// </summary>
+        /// <param name="command">
+        /// The command to search for.
+        /// </param>
+        /// <param name="provider">The provider that the command was found in.</param>
+        /// <returns>
+        /// Full path to the command.
+        /// </returns>
+        private string GetNextLiteralPathThatExists(string command, out ProviderInfo provider)
+        {
+            string resolvedPath = _context.LocationGlobber.GetProviderPath(command, out provider);
+
+            if (provider.Name.Equals("FileSystem", StringComparison.OrdinalIgnoreCase)
```

Comment:
Compare to [FileSystemProvider.ProviderName](https://github.com/PowerShell/PowerShell/blob/df3709409215c5c67f26dc41702d0d0830b57104/src/System.Management.Automation/namespaces/FileSystemProvider.cs#L73)